lab01-util/build/reports/tests/test/classes/by.artempvn.lab01.util.StringUtilsTest.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
```

Comment:
Please add `build` folder to .gitignore. 
Only source code should be presented in repository.